pkg/virt-controller/watch/export/export.go

```diff
@@ -111,6 +112,8 @@ const (
 
 	exporterPodFailedOrCompletedEvent = "ExporterPodFailedOrCompleted"
 	exporterPodCreatedEvent           = "ExporterPodCreated"
+	vmStartedEvent                    = "ExportedVMStarted"
```

Comment:
hmm, maybe generic `ExportPaused` with more detail in message

Reply:
Done, if I understood you correctly.